src/test/java/org/tests/cache/TestCacheCollectionIds.java

```diff
@@ -79,7 +79,7 @@ private int fetchCustomer(Integer id) {
     Customer customer2 = Ebean.find(Customer.class, id);
 
     List<Contact> contacts2 = customer2.getContacts();
-    contacts2.size();
```

Comment:
This could have side effects (but shouldn't) as "BeanList.size()" will trigger a lazy load. But BeanList.iterator() does the same.